src/Adapter/CombinationDataProvider.php

```diff
@@ -69,6 +69,30 @@ public function getFormCombination($combinationId)
         );
     }
 
+    public function getFormCombinations(array $combinationIds, int $languageId)
```

Comment:
Could you please add Phpdoc for this method? Otherwise it's kind of had to know what its purpose is.